### PR TITLE
Fix wrong service invocation type in http tracing

### PR DESCRIPTION
**Pull request by RayOnFire**
# Description

If request contains header `dapr-app-id`, this PR will treat it as a service invocation.

## Issue reference

Please reference the issue this PR will close: #_[7085]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by ItalyPaleAle** [comment]
Thanks for this!

I believe when #6919 is merged, it should supersede this issue and fix this bug too. Let's get that merged first :)

**Review by ItalyPaleAle — CHANGES_REQUESTED** [review]
Let’s wait on #6919 which should address this issue at the root.

**Comment by yaron2** [comment]
@RayOnFire if you can please pull the master branch version, build a `daprd` binary and see if the issue is now resolved, that would help a lot.

File: pkg/diagnostics/http_tracing.go [hunk 1]
```diff
@@ -206,9 +206,16 @@ func spanAttributesMapFromHTTPContext(rw responsewriter.ResponseWriter, r *http.
 	statusCode := rw.Status()
 
 	m := map[string]string{}
-	_, componentType := getAPIComponent(path)
 
 	var dbType string
+	var componentType string
+
+	appId := r.Header.Get("dapr-app-id")
```
Comment (daixiang0) [inline pkg/diagnostics/http_tracing.go:213]:
```suggestion
	appID := r.Header.Get("dapr-app-id")
```

**Comment by ItalyPaleAle** [comment]
In the current master branch, we changed how tracing works significantly. Could you please make sure this PR is still needed?

**Comment by yaron2** [comment]
Ping @RayOnFire

**Comment by yaron2** [comment]
Ping @RayOnFire

**Comment by RayOnFire** [comment]
Hi, sorry for the late reply. I think in the current master branch, this PR is no longer needed. Thank you for your comment & code review!

**Pull request closed without merging**